docs/sbctl.8.txt

```diff
@@ -113,12 +113,28 @@ EFI binary commands
                 Sign all the generated bundles.
 
 **remove-bundle** <NAME>::
-        Removes a bundle from the list.
+        Removes a bundle from the list. This does not delete the bundle itself.
 
 **list-bundles**::
         List all registered bundles to generate.
 
 
+Bundles
+-------
+
+Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)
```

Comment:
`EFI executables` maybe? A little bit unsure if "binaries" convey the same meaning? I also think we should write `(initramfs, kernel and cmdline)`.